### PR TITLE
Pinch to zoom

### PR DESCRIPTION
**Pull request by oblador**
Based loosely on @ZackLeonardo's PR #85 for use in upcoming 1.0 refactor.

### CODE REVIEW

File: src/ZoomableImage.android.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  };
+
+  render() {
+    return (
+      <View
+        style={this.props.style || styles.container}
+        onLayout={this.handleLayout}
+        {...this.panResponder.panHandlers}
+      >
+        <Animated.Image
```
Comment (peteroid) [inline src/ZoomableImage.android.js]: Should we use `Animated.View` instead so that we can easily support custom Image component ?

Comment (oblador) [reply src/ZoomableImage.android.js]: Good point, I've refactored to be agnostic of underlying component.

**Comment by peteroid** [comment]
Do you think we should add `useNativeDriver` whenever possible ? Or we make it to be a prop that others can override it if they don't want it. Actually, `useNativeDriver` prop could be added to any `Animated` in this repo for the sake of performance.

**Comment by oblador** [comment]
@peteroid Fixed, although in this specific case I don't think it makes much of a difference. Where it would make a difference to use the native driver is the gesture handler, but since it's kind of complex I don't think we can do so easily. To make optimal performance we'd need to tap into native code or wait for `react-native-gesture-handler` to be merged into `react-native` master. For iOS we get this for free with the `ScrollView`

**Comment by oblador** [comment]
@peteroid You OK to merge?

**Comment by peteroid** [comment]
It looks good. Let me try it out in my own project.

**Review by peteroid — APPROVED** [review]

**Comment by peteroid** [comment]
Just a side note, if I am using this as below, I need to [forward the native props](https://facebook.github.io/react-native/docs/direct-manipulation.html#forward-setnativeprops-to-a-child) from the scrollview to make it render without error. I am not sure if this an issue or not.
```
<Lightbox>
  <Zoomable>
    <MyComponent />
  </Zoomable>
</Lightbox>
```

**Comment by azhararmar** [comment]
Is there no way to have stable pinch to zoom feature? this is in so-many cases so useful.

**Pull request merged**